CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Guidelines for Contribution
+
+<br>
+
+## How can I contribute in vscode-seneca-college project?
+
+<br>
+
+Never contributed to open source before? Are you curious in how contributions function in our project?
+Here is a brief summary!
+
+<br>
+
+## **Here is a brief overview of how to contribute to vscode-seneca-college:**
+
+---
+
+<br>
+
+1. Start by identifying an issue or a feature that you would want to add.
+
+<br>
+
+2. In your local GitHub organization, fork the repository linked to the problem. The repository `underyour-GitHub-username/repository-name` will now be accessible to you.
```

Comment:
Most probably a personal user account will be used rather than an organization, so can be changed to "Fork current repository to produce a personal copy of this project by clicking the "Fork" button in the upper right corner of your screen. If you leave the default settings, the repository "your-username/vscode-seneca-college" will now be appear under your profile."